src/ServiceControl.Monitoring.SmokeTests.LegacyAzureServiceBus/packages.config

```diff
@@ -5,7 +5,7 @@
   <package id="Newtonsoft.Json" version="11.0.2" targetFramework="net46" />
   <package id="NServiceBus" version="7.1.4" targetFramework="net46" />
   <package id="NServiceBus.AcceptanceTesting" version="7.1.4" targetFramework="net46" />
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="8.0.3" targetFramework="net46" />
+  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="8.0.6" targetFramework="net461" />
```

Comment:
Out of curiosity, what caused the bump from net46 to net461? Was that something we had changed in `NServiceBus.Azure.Transports.WindowsAzureServiceBus` ?

Reply:
That is a good question, don't know. I think it was the tooling of VS. Downgrading and see whether it works

Reply:
Sounds reasonable, the project is targeting 461 so I guess references you touch would get updated?

https://github.com/Particular/ServiceControl.Monitoring/blob/develop/src/ServiceControl.Transports.LegacyAzureServiceBus/ServiceControl.Transports.LegacyAzureServiceBus.csproj#L4